packages/server/routes/api/guilds.js

```diff
@@ -47,31 +48,163 @@ router.get("/", AuthController.authenticate, async (request, response) => {
   }
 });
 
+// Get guild by ID
 router.get(
+  "/:guildId",
+  AuthController.authenticate,
+  async (request, response) => {
+    try {
+      const guildId = request.params.guildId;
+      const guild = await GuildController.getGuild(guildId);
+
+      return response.status(200).json({
+        status: "success",
+        data: {
+          guild,
+        },
+      });
+    } catch (error) {
+      if (error.name === "PrismaClientKnownRequestError") {
```

Comment:
I did some more looking into [Prisma docs](https://www.prisma.io/docs/orm/prisma-client/debugging-and-troubleshooting/handling-exceptions-and-errors) and they recommend using `error instanceof PrismaClientKnownRequestError` to check if the error is an instance of one of the Prisma error subclasses instead of checking `error.name` but you guys got the general idea down with error handling.